modules/context/context.go

```diff
@@ -265,6 +267,12 @@ func (ctx *Context) ServerError(logMsg string, logErr error) {
 func (ctx *Context) serverErrorInternal(logMsg string, logErr error) {
 	if logErr != nil {
 		log.ErrorWithSkip(2, "%s: %v", logMsg, logErr)
+		if errors.Is(logErr, &net.OpError{}) {
```

Comment:
```suggestion
		if errors.Is(logErr, net.OpError) {
```
shouldn't work this too ?

Reply:
That won't compile. 

net.OpError is a type not an error, (net.OpError doesn't implement error either, *net.OpError would be the type that implements the error interface.)

Reply:
For what's worth: go internally also use this error as a pointer: https://github.com/golang/go/blob/2ebe77a2fda1ee9ff6fd9a3e08933ad1ebaea039/src/internal/poll/error_test.go#L38 Approving.